DuckDuckGo/TabSwitcherButton.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  TabSwitcherButton.swift
+//  DuckDuckGo
+//
+//  Copyright © 2017 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import UIKit
+import Lottie
+
+protocol TabSwitcherButtonDelegate: NSObjectProtocol {
+    
+    func showTabSwitcher()
+    
+}
+
+class TabSwitcherButton: UIView {
+    
+    struct Constants {
+        
+        static let fontSize: CGFloat = 10
+        static let fontWeight: CGFloat = 5
+        static let maxTextTabs = 100
+        static let labelFadeDuration = 0.3
+        static let buttonTouchDuration = 0.2
+        static let tintAlpha: CGFloat = 0.5
+        
+    }
+    
+    weak var delegate: TabSwitcherButtonDelegate?
+    
+    let tint = UIView(frame: .zero)
+    let anim = LOTAnimationView(name: "new_tab")
+    let label = UILabel()
+    
+    var tabCount: Int = 0 {
+        didSet {
+            if tabCount == 0 {
+                label.text = nil
+                return
+            }
+            
+            if tabCount >= Constants.maxTextTabs {
```

Comment:
This isn't rendering quite right. As it is no longer an image, can we render it in the attirbuted string below.
<img width="361" alt="max_text_tabs" src="https://user-images.githubusercontent.com/1652407/44526177-1549b200-a6db-11e8-913f-117f2846fa7b.png">